### ISSUE TITLE
Crash of recent VCS checkouts as well as 0.9.0 upon adding a city

### ISSUE THREAD

**Issue by pmattern** [dglent/meteo-qt#50]
Recent VCS checkouts as well as release 0.9.0 are crashing on new user profiles on Arch Linux as soon as a city got selected.

To reproduce start meteo-qt on a user profile where it hadn't been used before or where `$XDG_CONFIG_HOME/meteo-qt` had been deleted and select an arbitrary city from dialogue "meteo-qt". As soon as a city was selected that way and `OK` is clicked in dialogue "Meteo-qt Configuration" the application crashes.
When launched from a virtual terminal there's output
```
Traceback (most recent call last):
File "/usr/lib/python3.5/site-packages/meteo_qt/meteo_qt.py", line 557, in config
self.config_save()
File "/usr/lib/python3.5/site-packages/meteo_qt/meteo_qt.py", line 543, in config_save
self.tray()
File "/usr/lib/python3.5/site-packages/meteo_qt/meteo_qt.py", line 433, in tray
temp_decimal = eval(self.settings.value('Decimal'))
TypeError: eval() arg 1 must be a string, bytes or code object
```
on the shell, traces can be found at https://gist.github.com/pmattern/0290959a4e2563720b019fbb4309c682. Problem is not related to #44, bisecting suggests it's a regression introduced in 8d0716d.

**Comment by dglent** [comment]
Thanks for the report
I have fixed some other errors making the application crashing on new user profiles:
https://github.com/dglent/meteo-qt/commit/5a7953f9c4f7188eddbb2bc305f87745d1ea083c#diff-5ed3175115338b6f48aaa9ff69a82ab4

But the crash you mention is already fixed in a previous commit, before the 0.9.0:
https://github.com/dglent/meteo-qt/commit/77892b9c1462abb0b34d60583e4d440a21a65b0c

I will release a 0.9.1 with the above fixes although

**Comment by pmattern** [comment]
The crashes addressed in this issue have indeed gone as of release 0.9.1 / 6c6c1f6.
This
> But the crash you mention is already fixed in a previous commit, before the 0.9.0:
> 77892b9

isn't plausible, though. The crash can be seen running all commits from 8d0716d to 4432309.

Side-note on tagging:
As of 6c6c1f6 `git describe` still yields output like this
```
$ git describe --always 
v0.8.1-81-g6c6c1f6
```
which is far behind the actual tag `0.9.1`.
To my knowledge this is due to using so-called "lightweight" tags which don't get considered by `git describe`. While it probably doesn't make sense to fiddle with tags that have already been set you may want to use so-called "annotated" tags next time, e. g. by running `git tag -a <some tag>`. That way `git describe` should yield more reasonable output like `v0.9.2-10-g1234abc` again.
